Project3/style.css

```
@@ -0,0 +1,152 @@
+figure {
+    position: relative;
+
+
+}
+
+figure img {
+    transition: all 1s;
+    width: 200px;
+    height: auto;
+    margin: 0%;
+}
+
+figure img:hover {
+    transform: scale(1.3);
+}
+
+figcaption {
+    display: none;
```

See note below about adding `color` to this selector instead of `:target`
```suggestion
    color: gb(255, 255, 255);
    display: none;
```